install.sh

```diff
@@ -20,5 +20,5 @@ fi
 sudo apt-get update
 sudo apt-get install libudev-dev libusb-1.0
 sudo apt-get install -y gettext
-pip install -r requirements.txt
+pip install .
```

Comment:
What's the reasoning behind this change as well?

Reply:
Sorry, I don't think this should have changed, I think I was debugging something on another PR - the only thing I can think of is that in the pipeline for `pyproject.toml` there will be `requirements.txt` and a new `optional_requirements.txt` that some libraries will have for optional libraries (like how some of the libraries can use, but don't require, Pillow).  I think there may need to be a solution to that latter, but definitely changing `pip install -r requirements.txt` is not intended.  Sorry for letting it sneak in!